packages/ragbits-evaluate/src/ragbits/evaluate/dataset_generator/tasks/passages_gen_task.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any
```

Comment:
think that dataset_generator should have subpackages - like qa for now in the future maybe corpus, ner, etc.